src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,49 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotationss
+        type: array
+        items:
+            type: object
+            properties:
+                uri:
+                    description: Annotation URI
+                    type: string
+                    format: uri
+                title:
+                    description: Title
```

Comment:
Title of what? The annotation?

Reply:
This would be the title of the page that has been annotated.